src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -67,6 +67,14 @@
                 localStorage.setItem(profilerStorageKey + name, value);
             },
 
+            addEvent = function (element, eventName, callback) {
+                if (document.all) {
```

Comment:
There's also a drawback, in ie8 and 9 the reference to this in the event handler refers to the document object instead of the caller object. So eventHandlers using this should have an alternative to the caller object